src/objects-table/ObjectsTable.js

```diff
@@ -421,7 +430,7 @@ class ObjectsTable extends React.Component {
                     ) : null}
                 </div>
 
-                {onCreate && <ListActionBar onClick={onCreate} />}
+                {onCreate && <ListActionBar onClick={onCreate} label={createButtonLabel} />}
```

Comment:
*Minor:* I know it's a breaking change (should be avoided if third parties used the library) but wouldn't it make sense:

```
{floatingButtonAction && <ListActionBar onClick={floatingButtonAction} label={floatingButtonLabel} />}
```